src/Microsoft.AspNetCore.Mvc.Testing/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -15,11 +15,38 @@
     <PreserveCompilationContext>true</PreserveCompilationContext>
   </PropertyGroup>
 
-  <Target Name="CopyAditionalFiles" AfterTargets="Build" Condition="'$(TargetFramework)'!=''">
+    <Target Name="_ResolveMvcTestProjectReferences" DependsOnTargets="ResolveReferences">
```

Comment:
I'm not an MSBuild expert so I don't think I can provide valuable feedback on this file. But we have an expert on the team who goes by @natemcmaster . Nate, can you have a look at this MSBuild code with @javiercn and review? (Or, if you already have, thank you!)